scripts/zowe-api-mediation-configure.sh

```diff
@@ -178,6 +213,9 @@ cat <<EOF >$TEMP_DIR/orion.yml
 #
 services:
     - serviceId: orion
+      title: IBM z/OS Orion
+      description: IBM z/OS Orion Languages REST API service
+      catalogUiTileId: orion
```

Comment:
This shouldn't have a catalog UI tile

Reply:
It doesn't. It's the 
catalogUiTiles:
    zos:
        title: z/OS Miscellaneous services
        description: IBM z/OS Miscellaneous REST services

that does that. However the text is somewhat redundant though